src/cli/search.rs

```diff
@@ -99,11 +100,155 @@ pub async fn execute(args: Args) -> miette::Result<()> {
     let repo_data = fetch_sparse_repodata(&channels, &platforms).await?;
 
     let p = PackageName::try_from(package_name.clone()).into_diagnostic()?;
+
+    if contains_wildcard(&package_name) {
+        let result = search_package_by_wildcard(p, &package_name, repo_data, limit).await;
+        return match result {
+            Ok(_) => Ok(()),
+            Err(report) => Err(report),
+        };
+    }
+
+    let result = search_exact_package(p, &package_name, repo_data).await;
+    return match result {
+        Ok(_) => Ok(()),
+        Err(report) => Err(report),
+    };
```

Comment:
Use `let _ =` to ignore the `Ok` result while still propagating any `Err` with `?`.
This approach is useful when the function's success result isn't needed, but errors must be handled.
```suggestion
    if contains_wildcard(&package_name) {
        let _ = search_package_by_wildcard(p, &package_name, repo_data, limit).await?;
    } else {
        let _ = search_exact_package(p, &package_name, repo_data).await?;
    }
```